### ISSUE TITLE
Installing the package "vulkan-headers" required on Arch

### ISSUE THREAD

**Issue by albin-engstrom** [flightlessmango/MangoHud#42]
I needed to install the package "vulkan-headers" to get the hud working on Arch. Without it games seemed to freeze early in the starting process if the hud was enabled.

It's not one of the dependencies build.sh checks for, but I noticed the build process tries to compile it, so I assume it not being in the dependencies is intentional.

It looks like the compilation of it fails though, hence why installing the package was needed.
Here's what seems to be the relevant error:

`Cloning into '/home/sasamus/Downloads/Temporary/MangoHud/build/meson32/modules/Vulkan-Headers'...
[22/25] Compiling C++ object 'src/25a6634@@MangoHud@sha/overlay.cpp.o'.
../../src/overlay.cpp: In function ‘void init_gpu_stats(device_data*)’:
../../src/overlay.cpp:794:26: warning: unused variable ‘instance_data’ [-Wunused-variable]
  794 |    struct instance_data *instance_data = device_data->instance;
      |       `

I don't know if installing the package is a better solution than compiling it, but at least it seems to work fine for me.

This was tested with the latest master, which is at [this commit](https://github.com/flightlessmango/MangoHud/commit/54950853b50a8b63feddc56687a12e6f452c4fe4) as of writing this.

**Comment by jackun** [comment]
> It looks like the compilation of it fails though, 

It doesn't. It is just a warning of unused (for now) variable. `vulkan-headers` shouldn't affect anything, other than compilation. Maybe `vulkan-validation-layers` instead?

**Comment by albin-engstrom** [comment]
Yeah, I wasn't quite sure if the error/warning was significant. My C++ is very rusty. But as `vulkan-headers` seemed to be needed I thought it might be related. 

But as you said, after some further testing it seems like `vulkan-headers` was not related to my issues. After uninstalling it the hud still works, even after uninstalling and recompiling/installing MangoHud.

The reason I got onto the `vulkan-headers` trail was that when I had issues I decided to try to install a [MangoHud AUR package](https://aur.archlinux.org/packages/mangohud) to see how it goes. That package installed `vulkan-headers` as a missing dependency and the hud worked.

Then I unistalled the AUR package and tried compiling MangoHud again and that now worked, the only difference as far as I could tell being that `vulkan-headers` was installed.

But as that does not seem to be it the only thing I can think of is that the AUR package does something that was needed for MangoHud to work that wasn't related to dependencies, something that isn't reverted when the package is uninstalled.

I looked over the PKGBUILD but nothing stood out, maybe someone more familiar with MangoHud can get something out of it.

As for 'vulkan-validation-layers', that package isn't and hasn't been installed at any point. So I don't think that's related either.
